src/_constants/ActionTypes.js

```diff
@@ -53,3 +53,7 @@ export const UPDATE_VIDEO_LIST = 'UPDATE_VIDEO_LIST';
 
 export const WATCHLIST_FAVOR_ASSET = 'WATCHLIST_FAVOR_ASSET';
 export const WATCHLIST_UNFAVOR_ASSET = 'WATCHLIST_UNFAVOR_ASSET';
+
+export const ADD_TOKEN = 'ADD_TOKEN';
```

Comment:
For these more 'deep' questions you can post on StackOverflow and ping dan_abramov on Twitter to answer. He said this is a good approach ;)